extensions-core/kafka-extraction-namespace/src/main/java/io/druid/query/lookup/KafkaLookupExtractorFactory.java

```diff
@@ -103,7 +103,7 @@ public KafkaLookupExtractorFactory(
       @JsonProperty("kafkaTopic") final String kafkaTopic,
       @JsonProperty("kafkaProperties") final Map<String, String> kafkaProperties,
       @JsonProperty("connectTimeout") @Min(0) long connectTimeout,
-      @JsonProperty("isOneToOne") boolean isOneToOne
+      @JsonProperty("isInjective") boolean isOneToOne
```

Comment:
Also, in `NamespaceLookupExtractorFactory` we use `"injective"` without the `is` prefix in the json property. Should we make it consistent?